autogen/main/variables.tf.tmpl

```diff
@@ -753,3 +753,11 @@ variable "enable_identity_service" {
 }
   {% endif %}
 {% endif %}
+
+{% if beta_cluster %}
```

Comment:
Any reason for restricting to beta? I see it available in the GA provider

https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#4310-aug-1-2022

Reply:
Hi Barath, 
Thanks for the feedback. No particular reason so I removed the restriction to beta in my last commit..